pulsar-discovery-service/src/main/java/com/yahoo/pulsar/discovery/service/server/DiscoveryServiceStarter.java

```diff
@@ -78,7 +78,7 @@ public void run() {
     protected static void startWebService(ServerManager server, ServiceConfig config) throws Exception {
         // add servlet
         Map<String, String> initParameters = new TreeMap<>();
-        initParameters.put("zookeeperServers", config.getZookeeperServers());
+        initParameters.put("localZookeeperServers", config.getLocalZookeeperServers());
```

Comment:
Also, we update the docs at https://github.com/yahoo/pulsar/blob/master/docs/ClusterSetup.md#service-discovery to add the newly introduced keys.